mmv1/third_party/terraform/services/compute/resource_compute_router_peer.go.erb

```diff
@@ -699,7 +757,12 @@ func resourceComputeRouterBgpPeerUpdate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
-	return resourceComputeRouterBgpPeerRead(d, meta)
+	err = resourceComputeRouterBgpPeerRead(d, meta)
+	if err != nil {
+		return err
+	}
+
+	return d.Set("md5_authentication_key", []interface{}{md5AuthenticationKeyProp})
```

Comment:
We always want to end by returning resourceComputeRouterBgpPeerRead, can you move this up?

Reply:
Done